arviz/plots/densityplot.py

```diff
@@ -171,10 +171,9 @@ def plot_density(
     for plotters in to_plot:
         length_plotters.append(len(plotters))
         for var_name, selection, _ in plotters:
-            label = make_label(var_name, selection)
+            label = make_label(var_name, selection).replace("\n", "_")
```

Comment:
I think it's better if `.replace` be done in `make_label`. perhaps by passing in a tuple. That way all the label making logic is in one place and `make_label` provides the finished label. 

Thoughts?

Reply:
Will fix on another PR per discussion. Thanks ari :)